openlibrary/templates/lists/widget.html

```diff
@@ -11,651 +11,652 @@
 $ container_id = 'widget-lists-wrapper-%s' % str(uid)
 
 <div id="$container_id" class="lists-widget-container">
-$if "lists" not in ctx.features:
```

Comment:
There is no need to change the indentation of this code. Please remove from your pull request. `git blame` is really useful when it comes to our code given the age of it, and I'd rather retain an easy glance at the history of this file. That's more important than the indentation level here.

The indentation also makes it a lot harder to review this patch as I have to manually investigate what has actually changed in the final HTML.